src/test/java/com/github/javafaker/InternetTest.java

```diff
@@ -276,4 +277,16 @@ public void testFarsiIDNs() {
         assertThat(f.internet().safeEmailAddress(), not(isEmptyOrNullString()));
         assertThat(f.internet().url(), not(isEmptyOrNullString()));
     }
+
+    @Test
+    public void testUserAgent() {
+        Internet.USER_AGENT[] agents = Internet.USER_AGENT.values();
+        for(Internet.USER_AGENT agent : agents){
+            assertThat(faker.internet().userAgent(agent), IsNot.not(isEmptyOrNullString()));
+            assertThat(faker.resolve("internet.user_agent." + agent.toString()), IsNot.not(isEmptyOrNullString()));
```

Comment:
I don't think you need this assert here. You're testing that the resolve method works and that the yml is correctly populated. I feel like your assert above should cover that.

Reply:
So you mean the second test is unnecessary ?